### ISSUE TITLE
Textboxes / popus of an app shown in fullscreen lead to undesired effects

### ISSUE THREAD

**Issue by cbtum** [ianyh/Amethyst#1070]
**Describe the bug**
Adobe Reader shows a PDF with annotations. When hovering these annotations, the window of adobe reader gets pushed to the left side of my screen. The annotation is only a very small textbox/popup but still gets half of the entire screen. Disabling Amethyst solves the problem. 

**Applications:**
Adobe Reader
Amethyst in fullscreen mode

**To Reproduce**
Open a PDF with annotations, comments etc in Adobe Reader.
Hover over some of these annotations (adobe reader will show some additional textbox/popup) with the corresponding comment.
The window of adobe reader gets pushed to the left half of the screen. The tiny little annotation gets the other half of the screen

**Expected behavior**
Popup / annotation textbox is just another window within adobe reader (which is still shown as fullscreen).

**Versions:**
 - macOS: 10.15.7
 - Amethyst: 0.15.4

**Comment by freebrowser1** [comment]
Make this 'float by default' at least optional, then other users who don't want this are not forced to use floating windows always.

And with the current versio you can add Adobe Reader to the list of Floating apps, just like I've done for some apps like in the screenshot.
![Screenshot 2020-11-20 at 08 11 02](https://user-images.githubusercontent.com/26625141/99770757-43abd700-2b08-11eb-946f-44fa4d7dde50.png)

**Comment by ianyh** [comment]
Yes if you want something to float by default you can add the application to the floating list and specify window titles. This is a little arcane, though, and Adobe Reader might be a case where we can improve the logic to avoid types of dialogs and popups so I will leave this open.

**Comment by freebrowser1** [comment]
There is only one issue: it should have an app identifier.
I have scrcpy which allows controlling the Android phone from the computer (Macbook in my case). However 'scrcpy' should be started from a terminal but has a window. Filling in the path of scrcpy such as '/opt/homebrew/bin/scrcpy' does not work. Can that be enabled ?

**Comment by cbtum** [comment]
Well basically, I would love to use "automatically float all applications except those listed" (without adding any application to the list) and use amethyst only on demand, i.e. when I press a shortcut.

However, once I select "automatically float all applications except those listed", the shortcuts for fullscreen, column etc are no longer working...
